### PR TITLE
[Integration Test][idrac_secure_boot]- Complete feature

### PR DESCRIPTION
**Pull request by Saksham-Nautiyal**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Review by ShivamSh3 — DISMISSED** [review]

**Review by ABHISHEK-SINHA10 — CHANGES_REQUESTED** [review]
Add scenario when restart is false combination with job_wait and job_wait_timeout

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

**Comment by ABHISHEK-SINHA10** [comment]
> Add scenario when restart is false combination with job_wait and job_wait_timeout

I checked once again and the scenario is available in one of the test files.

File: tests/integration/targets/idrac_secure_boot/tests/invalid_common.yaml [hunk 1]
```diff
@@ -78,7 +78,7 @@
           - invalid_ca_path.failed
           - not invalid_ca_path.changed
           - 'invalid_ca_path.msg == "[Errno 2] No such file or
-            directory: b''invalid''"'
+            directory"'
```
Comment (rajshekarp87) [inline tests/integration/targets/idrac_secure_boot/tests/invalid_common.yaml:81]: @Saksham-Nautiyal, Kindly check this change

Comment (Saksham-Nautiyal) [reply tests/integration/targets/idrac_secure_boot/tests/invalid_common.yaml:81]: Yes, Ansible is throwing this error msg only.

**Review by rajshekarp87 — APPROVED** [review]

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Pull request merged**